### PR TITLE
M2-2945: Summary tab: wrong calculation of Total Average value if lookup table is added to Subscale

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by muhurov — APPROVED** [review]

File: src/shared/utils/isSystemItem.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { LookupTableItems } from 'shared/consts';
+
+export const isSystemItem = (name: string) =>
```
Comment (alex-i-e) [inline src/shared/utils/isSystemItem.ts:3]: could please remove `isSystemItem` from src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscalesConfiguration.utils.tsx
and use only from one place?

**Review by alex-i-e — CHANGES_REQUESTED** [review]
could please remove isSystemItem from src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscalesConfiguration.utils.tsx
and use only from one place?

File: src/shared/utils/isSystemItem.ts (hunk 1 above)
Comment (nmialik) [reply src/shared/utils/isSystemItem.ts:3]: it has been already removed

Comment (alex-i-e) [reply src/shared/utils/isSystemItem.ts:3]: yeap, I missed it, apparently my eyes are tired at the end of the working day :)

**Pull request merged**